src/main/kotlin/frc/robot/Constants.kt

```
@@ -13,6 +21,49 @@ private val SPEAKER_POSE_BLUE = Translation2d(0.0, 5.5479442)
 val SPEAKER_POSE: Translation2d
     get() = SPEAKER_POSE_BLUE.flipIfNeeded()
 
+val choreoPoses: Map<String, Pose2d>
+    get() {
```

I don't know how expensive the parsing is. Why not parse it once on robot startup?

---

8ff98e74da6358997decd5416bd8b7e4a07db1a4

---

I think it's better to throw an exception:
1. I'd rather find out before the match we have a problem with auto alignment (and maybe with auto period as well).
2. This will crash the first time w euse the auto alignment mid-match, because the code will try to access a non-existent field in the map.

---

969ac656be12ef2d852b143a00f3bcb5f6b92760